gulpfile.js

```diff
@@ -161,6 +162,7 @@ gulp.task('subjects', function (done) {
 
     return gulp.src('./src/images/subjects/*.svg')
         .pipe(svgSprite(config))
+        .pipe(soften(2))
```

Comment:
Yeah but we are checking there other thinks like selector names (they are generated using filenames). This issue will solve itself when we will move to SVG icons.